modules/apps/object/object-service/src/main/java/com/liferay/object/internal/entry/util/ObjectEntryUtil.java

```diff
@@ -265,6 +266,13 @@ private static Map<String, Object> _toDTO(
 				jsonObject.put(entry.getKey(), entry.getValue());
 			}
 
+			Map<String, Serializable> attributes =
+				ObjectEntryThreadLocal.ActualExpandoValue.getExpandoValues();
+
+			ObjectEntryThreadLocal.ActualExpandoValue.clear();
+
+			jsonObject.put("customFields", attributes);
+
```

Comment:
@JhossephG you should only do this for the original DTO [here](https://github.com/liferay-bpm/liferay-portal/blob/032f0e5427744cd7490a290000ac6b66c242b2a2/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/entry/util/ObjectEntryUtil.java#L105-L113)